src/api/common/schemas/get-ledger.json

```diff
@@ -14,7 +14,12 @@
     "parentLedgerHash": {"$ref": "hash256"},
     "parentCloseTime": {"type": "integer", "minimum": 0},
     "totalDrops": {"$ref": "value"},
-    "transactionHash": {"$ref": "hash256"}
+    "transactionHash": {"$ref": "hash256"},
+    "transactions": {"type": "array", "items": {"type": "object"}},
+    "rawTransactions": {"type": "string"},
+    "transactionHashes": {"type": "array", "items": {"$ref": "hash256"}},
+    "rawAccounts": {"type": "string"},
```

Comment:
With regard to stringifying, it's not like the Paths in the transactions that just get passed around, and are supposed to be treated as an opaque string. 

If one of the design goals is that Most RippleAPI users can be oblivious to rippled json formats, they simply don't need to see the raw versions so perhaps their inclusion should be opt in.  And if they have opted in, it doesn't make much sense to stringify it IMO. Anyone who wants to do something with the rawAccounts will have to parse that (probably big) string.